airbyte-webapp/src/packages/cloud/locales/en.json

```
@@ -158,6 +158,12 @@
   "firebase.auth.error.default": "Confirmation email cannot be sent. Please try again later.",
 
   "signup.password.minLength": "Password should be at least 12 characters",
+  "signup.details.noCreditCard": "No credit card required",
+  "signup.details.instantSetup": "Instant setup",
+  "signup.details.freeTrial": "14-day free trial",
+  "signup.title": "Create your Airbyte account",
+  "signup.method.email": "Sign up with email",
+  "signup.method.oauth": "Sign up with Google or Github",
```

```suggestion
  "signup.method.email": "Sign up using email",
  "signup.method.oauth": "Sign up using Google or Github",
```

I think "sign up with x" sounds slightly off in English, but cc @nataliekwong for her opinion on this.